inspirehep/modules/references/processors.py

```diff
@@ -23,13 +23,66 @@
 """References Processors."""
 
 import re
+import six
 
 import idutils
 from isbn import ISBNError
 
 from inspirehep.utils.pubnote import split_pubnote
 
 
+# Matches any separators for author enumeratoins.
+RE_SPLIT_AUTH = re.compile(r',? and |,?\s*&|,|et al\.?|\(?eds?\.\)?',
```

Comment:
Isn't this missing some cases? Like, arbitrary spaces after the comma but `and` instead of `&`?